### ISSUE TITLE
Incompatibility between Django and HTMLHint extensions

### ISSUE THREAD

**Issue by AAkhilleus** [vscode-django/vscode-django#25]
First off, great extension so thanks for your effort.
I had installed previously HTMLHint extension in VSC and it worked fine, its feature on autocomplete/autoclosing attributes (example: with "`<header`" it generates `"<header></header>"`), but after installing your extension, this feature is no more.
it'd be great if both extensions worked together, or if your extension had that feature for Django and HTML attributes.
Thanks in advance

**Comment by inickua** [comment]
Yes, I support it too, I have the same problems, your expansion is good, but it breaks the functionality of other extensions.

**Comment by Harryki** [comment]
i have same problem too, looking for solution for this
